web/src/cypress/support/select-by-label.ts

```diff
@@ -30,59 +41,65 @@ function selectByLabel(sub: any, label: string): Cypress.Chainable {
     .should('not.exist')
 }
 
-function findByLabel(sub: any, label: string): Cypress.Chainable {
-  return cy.wrap(sub).then(el => {
-    const isSearchSelect =
-      el.parents('[data-cy=material-select]').data('cy') === 'material-select'
-    if (isSearchSelect) {
-      cy.wrap(sub)
-        .parents('[data-cy=material-select]')
-        .should('have.attr', 'data-cy-ready', 'true')
-        .find('[data-cy=search-select-input]')
-        .children()
-        .last() // skip the chips
-        .children()
-        .last() // ignore the clear button
-        .find('svg') // drop-down icon
-        .should('have.length', 1)
-        .should('be.visible')
-        .click()
-        .should('not.have.focus')
-
-      cy.focused()
-        .should('be.visible')
-        .type(label)
-
-      cy.get('[data-cy=select-dropdown]').should('not.contain', 'Loading')
-
-      return cy
-        .get('[data-cy=select-dropdown]')
-        .contains('[role=menuitem]', label)
-    }
+function clearSelect(sub: any): Cypress.Chainable {
+  return cy.wrap(sub)
+    .parents('[data-cy=material-select]')
+    .should('have.attr', 'data-cy-ready', 'true')
+    .find('[data-cy=search-select-input]')
+    .children()
+    .last() // skip the chips
+    .children()
+    .first() // get the clear button
+    .find('svg') // clear field icon
+    .should('have.length', 1)
+    .should('be.visible')
+    .click()
+}
 
+function findByLabel(sub: any, label: string): Cypress.Chainable {
+  if (isSearchSelect(sub)) {
     cy.wrap(sub)
-      .parent()
-      .find('[role=button]')
+      .parents('[data-cy=material-select]')
+      .should('have.attr', 'data-cy-ready', 'true')
+      .find('[data-cy=search-select-input]')
+      .children()
+      .last() // skip the chips
+      .children()
+      .last() // ignore the clear button
+      .find('svg') // drop-down icon
+      .should('have.length', 1)
+      .should('be.visible')
       .click()
+      .should('not.have.focus')
```

Comment:
This is similar code but instead of `first()` to click the clear svg, it's using `last()` to click the dropdown svg and then move focus to the menu items

Reply:
We should be able to alias the first half of the gets to reduce on code there, if we want

Reply:
ah, I didn't catch that, it's probably fine as is then if they are not the same